_layouts/base.html

```diff
@@ -19,7 +19,7 @@
     </script>
 
     <!-- Webfont -->
-    <link href='http://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
+    <link href='//fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
```

Comment:
Yes. `//` is scheme-relative address. If the page is served over http, it'll translate to http://... and if the current page is https, it'll translate to https://...